src/components/CandidateContest.tsx

```diff
@@ -109,31 +178,53 @@ const WriteInCandidateInput = styled.input.attrs({
 
 interface Props {
   contest: CandidateContestInterface
-  vote: Vote
+  vote: CandidateVote
   updateVote: UpdateVoteFunction
 }
 
 interface State {
   attemptedOverVoteCandidate: OptionalCandidate
   candidatePendingRemoval: OptionalCandidate
+  isScrollAtBottom: boolean
```

Comment:
Please LMK if this can be more clear. What I intended this to mean is:

- `isScrollAtBottom`: The user has scrolled to the very bottom. In this state the button to scroll down is disabled.
- `isScrollAtTop`: This is the initial state or the state when the user has scrolled to the very top. In this state the button to scroll up is disabled and the shadow at the top is hidden.